chore(root): publish modules

pengyuc-bitgo opened the pull request:
@bitgo/abstract-cosmos@10.0.8

@bitgo/abstract-eth@21.0.0

@bitgo/abstract-utxo@8.1.1

@bitgo/account-lib@23.0.8

bitgo@38.2.1

@bitgo/blockapis@1.9.2

@bitgo/key-card@0.22.13

@bitgo/sdk-api@1.45.3

@bitgo/sdk-coin-ada@4.1.6

@bitgo/sdk-coin-algo@2.1.6

@bitgo/sdk-coin-arbeth@21.0.0

@bitgo/sdk-coin-atom@12.0.8

@bitgo/sdk-coin-avaxc@5.0.0

@bitgo/sdk-coin-avaxp@5.0.7

@bitgo/sdk-coin-bch@2.0.8

@bitgo/sdk-coin-bcha@2.0.8

@bitgo/sdk-coin-bera@2.0.8

@bitgo/sdk-coin-bld@2.0.8

@bitgo/sdk-coin-bsc@22.0.8

@bitgo/sdk-coin-bsv@2.0.8

@bitgo/sdk-coin-btc@2.0.8

@bitgo/sdk-coin-btg@2.0.8

@bitgo/sdk-coin-celo@3.0.0

@bitgo/sdk-coin-coreum@20.0.8

@bitgo/sdk-coin-cspr@2.0.8

@bitgo/sdk-coin-dash@2.0.8

@bitgo/sdk-coin-doge@2.0.8

@bitgo/sdk-coin-dot@4.0.8

@bitgo/sdk-coin-eos@2.1.1

@bitgo/sdk-coin-etc@2.0.8

@bitgo/sdk-coin-eth@24.0.0

@bitgo/sdk-coin-eth2@3.0.7

@bitgo/sdk-coin-ethw@20.0.8

@bitgo/sdk-coin-hash@2.0.8

@bitgo/sdk-coin-hbar@2.0.8

@bitgo/sdk-coin-injective@2.0.8

@bitgo/sdk-coin-islm@2.0.8

@bitgo/sdk-coin-ltc@3.0.8

@bitgo/sdk-coin-near@2.0.8

@bitgo/sdk-coin-opeth@18.0.0

@bitgo/sdk-coin-osmo@2.0.8

@bitgo/sdk-coin-polygon@20.1.1

@bitgo/sdk-coin-rbtc@2.0.8

@bitgo/sdk-coin-sei@2.0.8

@bitgo/sdk-coin-sol@4.0.7

@bitgo/sdk-coin-stx@3.1.3

@bitgo/sdk-coin-sui@5.0.8

@bitgo/sdk-coin-tia@2.0.8

@bitgo/sdk-coin-ton@2.0.8

@bitgo/sdk-coin-trx@2.0.8

@bitgo/sdk-coin-xlm@3.0.8

@bitgo/sdk-coin-xrp@2.0.8

@bitgo/sdk-coin-xtz@2.1.6

@bitgo/sdk-coin-zec@2.0.8

@bitgo/sdk-coin-zeta@2.0.8

@bitgo/sdk-coin-zketh@2.0.0

@bitgo/sdk-core@26.8.2

@bitgo/sdk-rpc-wrapper@2.0.8

@bitgo/sdk-unified-wallet@11.0.8

@bitgo/utxo-bin@2.47.10

@bitgo/utxo-ord@1.20.13

pengyuc-bitgo committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@10.0.7
 - @bitgo/abstract-eth@20.1.6
 - @bitgo/abstract-utxo@8.1.0
 - @bitgo/account-lib@23.0.7
 - bitgo@38.2.0
 - @bitgo/blockapis@1.9.1
 - @bitgo/express@9.62.2
 - @bitgo/key-card@0.22.12
 - @bitgo/sdk-api@1.45.2
 - @bitgo/sdk-coin-ada@4.1.5
 - @bitgo/sdk-coin-algo@2.1.5
 - @bitgo/sdk-coin-arbeth@20.0.7
 - @bitgo/sdk-coin-atom@12.0.7
 - @bitgo/sdk-coin-avaxc@4.0.6
 - @bitgo/sdk-coin-avaxp@5.0.6
 - @bitgo/sdk-coin-bch@2.0.7
 - @bitgo/sdk-coin-bcha@2.0.7
 - @bitgo/sdk-coin-bera@2.0.7
 - @bitgo/sdk-coin-bld@2.0.7
 - @bitgo/sdk-coin-bsc@22.0.7
 - @bitgo/sdk-coin-bsv@2.0.7
 - @bitgo/sdk-coin-btc@2.0.7
 - @bitgo/sdk-coin-btg@2.0.7
 - @bitgo/sdk-coin-celo@2.0.7
 - @bitgo/sdk-coin-coreum@20.0.7
 - @bitgo/sdk-coin-cspr@2.0.7
 - @bitgo/sdk-coin-dash@2.0.7
 - @bitgo/sdk-coin-doge@2.0.7
 - @bitgo/sdk-coin-dot@4.0.7
 - @bitgo/sdk-coin-eos@2.1.0
 - @bitgo/sdk-coin-etc@2.0.7
 - @bitgo/sdk-coin-eth@23.1.6
 - @bitgo/sdk-coin-eth2@3.0.6
 - @bitgo/sdk-coin-ethw@20.0.7
 - @bitgo/sdk-coin-hash@2.0.7
 - @bitgo/sdk-coin-hbar@2.0.7
 - @bitgo/sdk-coin-injective@2.0.7
 - @bitgo/sdk-coin-islm@2.0.7
 - @bitgo/sdk-coin-ltc@3.0.7
 - @bitgo/sdk-coin-near@2.0.7
 - @bitgo/sdk-coin-opeth@17.0.7
 - @bitgo/sdk-coin-osmo@2.0.7
 - @bitgo/sdk-coin-polygon@20.1.0
 - @bitgo/sdk-coin-rbtc@2.0.7
 - @bitgo/sdk-coin-sei@2.0.7
 - @bitgo/sdk-coin-sol@4.0.6
 - @bitgo/sdk-coin-stx@3.1.2
 - @bitgo/sdk-coin-sui@5.0.7
 - @bitgo/sdk-coin-tia@2.0.7
 - @bitgo/sdk-coin-ton@2.0.7
 - @bitgo/sdk-coin-trx@2.0.7
 - @bitgo/sdk-coin-xlm@3.0.7
 - @bitgo/sdk-coin-xrp@2.0.7
 - @bitgo/sdk-coin-xtz@2.1.5
 - @bitgo/sdk-coin-zec@2.0.7
 - @bitgo/sdk-coin-zeta@2.0.7
 - @bitgo/sdk-coin-zketh@1.9.12
 - @bitgo/sdk-core@26.8.1
 - @bitgo/sdk-lib-mpc@9.4.0
 - @bitgo/sdk-rpc-wrapper@2.0.7
 - @bitgo/sdk-test@8.0.12
 - @bitgo/sdk-unified-wallet@11.0.7
 - @bitgo/statics@48.8.0
 - @bitgo/unspents@0.47.1
 - @bitgo/utxo-bin@2.47.9
 - @bitgo/utxo-lib@9.36.0
 - @bitgo/utxo-ord@1.20.12
 - @bitgo/web-demo@1.29.7

pengyuc-bitgo committed:
chore(root): release express

released express v9.62.2

WP-1749

ajays97 commented:
Looks like the checks are failing due to some vulnerable dependencies.

aarvay reviewed (commented):
flushing, waiting for green build

pengyuc-bitgo committed:
Merge branch 'master' into rel/latest

pengyuc-bitgo committed:
chore(root): publish modules

 - @bitgo/abstract-cosmos@10.0.8
 - @bitgo/abstract-eth@21.0.0
 - @bitgo/abstract-utxo@8.1.1
 - @bitgo/account-lib@23.0.8
 - bitgo@38.2.1
 - @bitgo/blockapis@1.9.2
 - @bitgo/express@9.62.3
 - @bitgo/key-card@0.22.13
 - @bitgo/sdk-api@1.45.3
 - @bitgo/sdk-coin-ada@4.1.6
 - @bitgo/sdk-coin-algo@2.1.6
 - @bitgo/sdk-coin-arbeth@21.0.0
 - @bitgo/sdk-coin-atom@12.0.8
 - @bitgo/sdk-coin-avaxc@5.0.0
 - @bitgo/sdk-coin-avaxp@5.0.7
 - @bitgo/sdk-coin-bch@2.0.8
 - @bitgo/sdk-coin-bcha@2.0.8
 - @bitgo/sdk-coin-bera@2.0.8
 - @bitgo/sdk-coin-bld@2.0.8
 - @bitgo/sdk-coin-bsc@22.0.8
 - @bitgo/sdk-coin-bsv@2.0.8
 - @bitgo/sdk-coin-btc@2.0.8
 - @bitgo/sdk-coin-btg@2.0.8
 - @bitgo/sdk-coin-celo@3.0.0
 - @bitgo/sdk-coin-coreum@20.0.8
 - @bitgo/sdk-coin-cspr@2.0.8
 - @bitgo/sdk-coin-dash@2.0.8
 - @bitgo/sdk-coin-doge@2.0.8
 - @bitgo/sdk-coin-dot@4.0.8
 - @bitgo/sdk-coin-eos@2.1.1
 - @bitgo/sdk-coin-etc@2.0.8
 - @bitgo/sdk-coin-eth@24.0.0
 - @bitgo/sdk-coin-eth2@3.0.7
 - @bitgo/sdk-coin-ethw@20.0.8
 - @bitgo/sdk-coin-hash@2.0.8
 - @bitgo/sdk-coin-hbar@2.0.8
 - @bitgo/sdk-coin-injective@2.0.8
 - @bitgo/sdk-coin-islm@2.0.8
 - @bitgo/sdk-coin-ltc@3.0.8
 - @bitgo/sdk-coin-near@2.0.8
 - @bitgo/sdk-coin-opeth@18.0.0
 - @bitgo/sdk-coin-osmo@2.0.8
 - @bitgo/sdk-coin-polygon@20.1.1
 - @bitgo/sdk-coin-rbtc@2.0.8
 - @bitgo/sdk-coin-sei@2.0.8
 - @bitgo/sdk-coin-sol@4.0.7
 - @bitgo/sdk-coin-stx@3.1.3
 - @bitgo/sdk-coin-sui@5.0.8
 - @bitgo/sdk-coin-tia@2.0.8
 - @bitgo/sdk-coin-ton@2.0.8
 - @bitgo/sdk-coin-trx@2.0.8
 - @bitgo/sdk-coin-xlm@3.0.8
 - @bitgo/sdk-coin-xrp@2.0.8
 - @bitgo/sdk-coin-xtz@2.1.6
 - @bitgo/sdk-coin-zec@2.0.8
 - @bitgo/sdk-coin-zeta@2.0.8
 - @bitgo/sdk-coin-zketh@2.0.0
 - @bitgo/sdk-core@26.8.2
 - @bitgo/sdk-rpc-wrapper@2.0.8
 - @bitgo/sdk-test@8.0.13
 - @bitgo/sdk-unified-wallet@11.0.8
 - @bitgo/utxo-bin@2.47.10
 - @bitgo/utxo-ord@1.20.13
 - @bitgo/web-demo@1.29.8

pengyuc-bitgo committed:
chore(root): release express 9.62.3

released express v9.62.3

Ticket: WP-1749
